NEWS.md

```diff
@@ -1,3 +1,19 @@
+### v9.2.0 (2022-10-06)
+
+* Added ability to instrument ES Modules with the New Relic ESM Loader.
+  * [Example ESM application](https://github.com/newrelic/newrelic-node-examples/tree/main/esm-app)
+
+* Added support for custom ESM instrumentation.
+  * There is structure to registering custom ESM instrumentation.  Set the relative path to the instrumentation entry point via `api.esm.custom_instrumentation_entrypoint`
+  * [Sample custom ESM instrumentaiton entrypoint](https://github.com/newrelic/newrelic-node-examples/blob/main/esm-app/custom-instrumentation/index.js)
+  * All the `newrelic.instrument*` methods will still work except `newrelic.instrumentLoadedModule`.  This is because it was geared toward CommonJS modules. 
```

Comment:
```suggestion
  * All the `newrelic.instrument*` methods will still work except `newrelic.instrumentLoadedModule`.  This is because it is geared toward CommonJS modules. 
```

still geared